music/package.json

```diff
@@ -50,7 +52,7 @@
     "test": "ts-node node_modules/tape/bin/tape src/**/*_test.ts",
     "build-demos": "webpack --config ./webpack/demo.config.ts",
     "publish-demos": "yarn build-demos && mkdir -p ../docs/music/demos && cp demos/*.{js,html,mid,css} ../docs/music/demos",
-    "run-demos": "yarn build-demos && yarn serve-demos",
+    "run-demos": "webpack-dev-server --config ./webpack/demo.config.ts",
     "serve-demos": "http-server demos/",
```

Comment:
is 'serve-demos' still needed?

Reply:
probably not? I wouldn't use it, but I wasn't sure if others would rather serve the built bundles rather than rely on webpack.